test/views/MovementConfirmationViewSpec.scala

```diff
@@ -25,19 +26,19 @@ import views.html.movement_confirmation_page
 class MovementConfirmationViewSpec extends UnitViewSpec with Stubs with Injector {
 
   val movementConfirmationPage = new movement_confirmation_page(mainTemplate)
-  val arrivalConfirmationView = movementConfirmationPage("EAL")(request, messages)
-  val departureConfirmationView = movementConfirmationPage("EDL")(request, messages)
+  val arrivalConfirmationView = movementConfirmationPage(Arrival)(request, messages)
+  val departureConfirmationView = movementConfirmationPage(Departure)(request, messages)
 
   "Movement Confirmation Page" should {
 
     "have correct messages" in {
 
       val messages = instanceOf[MessagesApi].preferred(request)
 
-      messages("movement.EAL.confirmation") mustBe "Arrival confirmation"
-      messages("movement.EDL.confirmation") mustBe "Departure confirmation"
-      messages("movement.choice.EAL.label") mustBe "Arrival"
-      messages("movement.choice.EDL.label") mustBe "Departure"
+      messages("movement.arrival.confirmation") mustBe "Arrival confirmation"
```

Comment:
Please move this to test key presence not value of them

Reply:
Next PR, we need to change it for all view specs. We don't have helpers method for it in this repository so I don't want to start this now